pkgs/development/python-modules/tensorrt/default.nix

```diff
@@ -33,20 +33,15 @@ buildPythonPackage rec {
 
   sourceRoot = ".";
 
-  runtimeDependencies = [
-    cudaPackages.tensorrt
-  ];
-
-  propagatedBuildInputs = [
-    cudatoolkit
+  buildInputs = [
+    cudaPackages.cudatoolkit
```

Comment:
Can we use the new individual cudaPackages instead? We're trying to move away from `cudaPackages.cudatoolkit` as much as possible.

Reply:
Looks as if we don't even need anything from `cudatoolkit` in this package.  I removed it and rebuilt the package to see what `autoPatchelf` failed to find, and it built just find, and I'm not getting a different error when I try to use the `tensorrt` python module.